### ISSUE TITLE
Sparky V1.2, (master branch PR#5)

### ISSUE THREAD

**Issue by thomastech** [hogthrob/Sparky#1]
Hello hogthrob,

I cloned your refactor branch and tested it. There were some bugs in the voice announcements. For example wrong amps were announced, CurrentOn message was not spoken. I also updated the top of file header comments to reflect your V1.2 changes. 

I intended my changes to be a PR on your refactor branch. But my files were committed instead. Not sure where I went wrong and hope it hasn't caused any problems for you.

Long story short, please test my revisions. If they pass your validation then please resubmit the PR so I can commit the revised files.

- Thomas

**Comment by thomastech** [comment]
I made some changes that eliminate the second reboot when using serial monitor. The edited files are platformio.ini and Pulsewelder.cpp.

Is it Ok if I merge this update with your Refactor branch? 

- Thomas
